backend/apis/clients/follow-ups/patch-follow-up.api.js

```diff
@@ -89,52 +92,64 @@ app.patch("/api/clients/:clientId/follow-ups/:followUpId", (req, res) => {
 
     updateFollowUpSql += mysql.format(
       `SELECT GROUP_CONCAT(services.serviceName SEPARATOR ', ') services FROM services WHERE id IN (?);`,
-      [serviceIds]
+      [serviceIds.length > 0 ? serviceIds : null]
     );
 
     pool.query(updateFollowUpSql, (err, updateResult) => {
       if (err) {
         return databaseError(req, res, err);
       }
 
+      let title = `Client received follow up`;
+      if (serviceIds.length > 0) {
+        title += ` regarding ${
+          updateResult[updateResult.length - 1][0]["services"]
+        }`;
+      }
+
       let clientLogUpdate = insertActivityLogQuery({
         detailId: newFollowUp.id,
         clientId,
-        title: `Client received follow up regarding ${
-          updateResult[updateResult.length - 1][0]["services"]
-        }`,
+        title,
         description: null,
         logType: "follow-up",
         addedBy: user.id,
+        dateAdded: newFollowUp.dateOfContact,
```

Comment:
I changed the client logs to show the date of the follow up instead of the date that it was created.

Reply:
Oh yes, my bad. I meant to change that when I was working on the interactions api. Thanks for catching!